mgmt_risk/models/risk.py

```diff
@@ -9,4 +9,30 @@ class MgmtRisk(models.Model):
     _name = "mgmt.risk"
     _description = "Mgmt Risk"
 
-    name = fields.Char()
+    name = fields.Char(required=True)
+    description = fields.Text()
+    risk_owner_id = fields.Many2one('res.users', required=True)
+    asset_ids = fields.Many2many('mgmt.asset', required=True)
+    severity_id = fields.Many2one('mgmt.severity', required=True)
+    hazard_ids = fields.Many2many('mgmt.hazard', required=True)
+    revision_risk_ids = fields.One2many('mgmt.risk', 'head_risk_id')
+    head_risk_id = fields.Many2one('mgmt.risk')
+    probability_id = fields.Many2one('mgmt.probability', required=True)
+    risk_score = fields.Float(compute='_compute_risk_score', store=True)
+    color = fields.Integer(compute='_compute_color', store=True)
+    stage = fields.Selection([
+        ('identify', 'Identify'),
+        ('evaluate', 'Evaluate'),
+        ('mitigate', 'Mitigate')
+    ], required=True)
+
+
+    @api.depends('severity_id', 'probability_id')
+    def _compute_risk_score(self):
+        for record in self:
+            severity = record.severity_id.value if record.severity_id else 0
+            probability = record.probability_id.value if record.probability_id else 0
+            if severity + probability > 0:
```

Comment:
Please use formula via company record.

Reply:
@janikvonrotz I was not sure if the formula shown in ERD implies division between multiplication and sum, but considering the `mgmt_risk_formula` field is a selection between multiplication and sum, I altered the method to reflect that and removed the division. Please let me know if this is correct